static/js/redux/ui/modals/user_settings_modal.jsx

```diff
@@ -1,354 +1,359 @@
-        className="welcome-modal max-modal"
-        closeOnClick={false}
-        keyboard={false}
-        modalStyle={modalStyle}
-      >
-        <div className="modal-content">
-          <div className="modal-header">
-            <div className="pro-pic" style={{ backgroundImage: `url(${this.props.userInfo.img_url})` }} />
-            <h1>Welcome!</h1>
-            { !this.state.isSigningUp ? cancelButton : null }
-          </div>
-          <div className="modal-body">
-            <div className="preference cf">
-              <h3>What&#39;s your major?</h3>
-              <Select
-                name="form-field-name"
-                value={this.props.userInfo.major}
-                options={majors}
-                searchable
-                onChange={this.changeMajor}
-              />
-            </div>
-            <div className="preference cf">
-              <h3>What&#39;s your graduating class year?</h3>
-              <Select
-                name="form-field-name"
-                value={this.props.userInfo.class_year}
-                options={[
-                                    { value: 2017, label: 2017 },
-                                    { value: 2018, label: 2018 },
-                                    { value: 2019, label: 2019 },
-                                    { value: 2020, label: 2020 },
-                                    { value: 2021, label: 2021 },
-                                    { value: 2022, label: 2022 },
-                                    { value: 2023, label: 2023 },
-                ]}
-                searchable
-                onChange={this.changeClassYear}
-              />
-            </div>
-            { preferences }
-            {/* { !this.state.isSigningUp ? notifications : null } */}
-            { fbUpsell }
-            { tos }
-            { !this.state.isSigningUp ? deleteDropdown : null }
-            <div className="button-wrapper">
-              <button
-                className="signup-button" onClick={this.hide}
-              >Save
-              </button>
-            </div>
-          </div>
-        </div>
-      </Modal>
-    );
-  }
-}
-
-UserSettingsModal.propTypes = {
-  userInfo: SemesterlyPropTypes.userInfo.isRequired,
-  closeUserSettings: PropTypes.func.isRequired,
-  saveSettings: PropTypes.func.isRequired,
-  changeUserInfo: PropTypes.func.isRequired,
-  // tokenRegistered: PropTypes.bool.isRequired,
-  // unsubscribeToNotifications: PropTypes.func.isRequired,
-  // subscribeToNotifications: PropTypes.func.isRequired,
-  highlightNotifs: PropTypes.bool.isRequired,
-  isUserInfoIncomplete: PropTypes.bool.isRequired,
-  isSigningUp: PropTypes.bool.isRequired,
-  acceptTOS: PropTypes.func.isRequired,
-  setVisible: PropTypes.func.isRequired,
-  setHidden: PropTypes.func.isRequired,
-  deleteUser: PropTypes.func.isRequired,
-  isDeleted: PropTypes.bool.isRequired,
-};
-
-export default UserSettingsModal;
+/*

```

Comment:
Task 2 requires that you add a field to student/models.py and then create a Django migration so that we can save this value in the database. You aren't bringing any new information from the database here - we want you to practice and understand how React/Redux grab data from the backend.